resources.tf

```diff
@@ -74,15 +74,54 @@ resource "aws_security_group" "ssh_sg" {
   }
 }
 
-# bread & butter - this tells TF to provision/create the actual instance
-resource "aws_instance" "spels" {
-  count                       = "${length(matchkeys(values(local.lx_amis),keys(local.lx_amis),split(",", var.tfi_lx_instances)))}"
-  ami                         = "${element(matchkeys(values(local.lx_amis),keys(local.lx_amis),split(",", var.tfi_lx_instances)), count.index)}"
-  instance_type               = "${var.tfi_lx_instance_type}"
+# instance to build win standalone package
+resource "aws_instance" "win_builder" {
+  count                       = "${local.win_count_builder}"
+  ami                         = "${data.aws_ami.win08.id}"
+  instance_type               = "${local.win_builder_instance_type}"
+  key_name                    = "${aws_key_pair.auth.id}"
   iam_instance_profile        = "${var.tfi_instance_profile}"
+  vpc_security_group_ids      = ["${aws_security_group.winrm_sg.id}"]
+  user_data                   = "<powershell>${data.template_file.win_userdata_builder_common.rendered} ${data.template_file.win_userdata_builder_specific.rendered}</powershell>"
+  associate_public_ip_address = "${var.tfi_assign_public_ip}"
+  subnet_id                   = "${var.tfi_subnet_id}"
+
+  tags {
+    Name = "${local.resource_name}-builder"
+  }
+
+  timeouts {
+    create = "30m"
+  }
+
+  connection {
+    type     = "winrm"
+    user     = "${var.tfi_rm_user}"
+    password = "${random_string.password.result}"
+    timeout  = "30m"
+  }
+
+  provisioner "file" {
+    content     = "${data.template_file.win_build_test.rendered}"
+    destination = "C:\\scripts\\builder_test.ps1"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "powershell.exe -File C:\\scripts\\builder_test.ps1",
+    ]
+  }
+}
+
+# bread & butter - provision/create windows instance
+resource "aws_instance" "win" {
+  count                       = "${local.win_count_all_requests}"
+  ami                         = "${element(local.win_amis_all_requests, count.index)}"
+  instance_type               = "${var.tfi_win_instance_type}"
   key_name                    = "${aws_key_pair.auth.id}"
-  vpc_security_group_ids      = ["${aws_security_group.ssh_sg.id}"]
-  user_data                   = "${data.template_file.lx_userdata.rendered}"
+  iam_instance_profile        = "${var.tfi_instance_profile}"
+  vpc_security_group_ids      = ["${aws_security_group.winrm_sg.id}"]
+  user_data                   = "<powershell>${element(data.template_file.win_userdata_common.*.rendered, count.index)} ${data.template_file.win_userdata_specific.rendered}</powershell>"
```

Comment:
Moving the `<powershell/>` xml elements here is pretty brilliant, nice! :D

Reply:
I was trying all kinds of complicated things to get two template files together and then thought, wait a second... The tag was another "wait a sec" moment.